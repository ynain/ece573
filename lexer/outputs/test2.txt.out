Token Type: IDENTIFER
Value: A
Token Type: OPERATOR
Value: :=
Token Type: IDENTIFER
Value: B
Token Type: OPERATOR
Value: +
Token Type: INTLITERAL
Value: 4
