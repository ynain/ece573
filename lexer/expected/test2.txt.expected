Token Type: IDENTIFIER
Value: A
Token Type: OPERATOR
Value: :=
Token Type: IDENTIFIER
Value: B
Token Type: OPERATOR
Value: +
Token Type: INTLITERAL
Value: 4